charts/enterprise-kyverno-operator/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: enterprise-kyverno-operator
 description: Helm Chart for Enterprise Kyverno Operator
 type: application
-version: v0.1.0-rc.2
+version: v0.1.0-rc.3
 appVersion: v0.1.0-rc.1
```

Comment:
Do we need to change the `appVersion` as well?

Reply:
Actually, the new app version build was not ready at that time. Available now. Updated.